### ISSUE TITLE
Conflict with admin wordpress

### ISSUE THREAD

**Issue by hamedmoody** [yahoo/pure#573]
Hi i am using wordpress 4.5.2, when i click on screen setting in top of dashboard, not show screen setting. when i disable pure.css it fixed.
this is my duty that send this issue.
Thanks for your lightweigh css framework
Bye

**Comment by Edwardzyc** [comment]
Could you provide screenshots/more details

**Comment by hamedmoody** [comment]
i don't know what is wrong but i sent screenshots
1- admin dashboard in normal view
![base-dashboard](https://cloud.githubusercontent.com/assets/9475089/15627285/5d2c237a-24f4-11e6-95e2-4198c29d1187.JPG)
2-admin dashboard with Pure when click on screen optons. then help has been hide and options not show:
![dashboard-with-pure](https://cloud.githubusercontent.com/assets/9475089/15627294/8f44cb3c-24f4-11e6-951c-63fc14eb741e.JPG)
3-when Pure not load no problem accurd
![dashboard-without-pure](https://cloud.githubusercontent.com/assets/9475089/15627301/ea7c9444-24f4-11e6-89a7-4374e8bdd1c9.JPG)
Thanks for your attention
Good Luck

**Comment by 0aveRyan** [comment]
This issue seems to be an issue with the jQuery triggering the removal of a `.hidden` class containing the screen options or help tab content. The class is on `#contextual-help-wrap`. When Pure is loaded, this hidden isn't removed onClick. Tested this on vanilla, latest trunk wordpress. Really odd.

I was able to workaround by selectively enqueueing Pure on the pages I needed it and running jQuery in the footer to trigger removal of that class -- again. 

I've tested using Pure in the WordPress Admin pretty thoroughly. It's basically safe except for breaking the JavaScript powering those tabs.
